src/form-layout-grid.scss

```diff
@@ -198,6 +212,7 @@ $cols: 11;
     .#{$blockRow} {
       .#{$blockCol} {
         .fd-form-label {
+          padding-top: 0.5rem;
```

Comment:
Is it the same value for compact and cozy?

Reply:
@InnaAtanasova this value is for compact `min-width: 1024px`
![image](https://user-images.githubusercontent.com/3863691/125990718-6d99cccb-f604-48ce-8e4e-c4b425c3b378.png)